streamlit_web.py

```diff
@@ -382,13 +387,11 @@ def sentiment():
 
 
 def overall_senti(senti_df):
-    """page for overall senti"""
```

Comment:
Also, just note that this docstring is removed. I want to make sure this is not _accidentally_ removed.